### PR TITLE
TiCDC: add trouble shooting info about DDL failures (#5449)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #5449 to release-5.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-cn repo:
git pr https://github.com/pingcap/docs-cn/pull/5776
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-cn.git pr/5776:release-5.0-ef8ef4414154
```

---

### What is changed, added or deleted? (Required)

Added information on how to handle DDL failures after 4.0.11

### Which TiDB version(s) do your changes apply to? (Required)
4.0.11 or above，5.0.0-rc excluded.

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/ticdc/pull/1291
https://github.com/pingcap/ticdc/pull/1352

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -372,3 +372,28 @@ replica.fetch.max.bytes=2147483648
...
+
+```shell
+cdc cli changefeed update -c test-cf --pd=http://10.0.10.25:2379 --start-ts 415241823337054210
+
+cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
+```
+
+> **注意：**
+>
+> + 以上步骤仅适用于 TiCDC v5.0.0 及以上版本（不包括 v5.0.0-rc）。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:398]: This is confirmed with @liuzix.

**Comment by TomShawn** [comment]
The translation is in https://github.com/pingcap/docs/pull/5016.

**Review by yikeke — APPROVED** [review]

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -372,3 +372,28 @@ replica.fetch.max.bytes=2147483648
 # 消费者端的可读取的最大消息字节数
 fetch.message.max.bytes=2147483648
 ```
+
+## TiCDC 同步时，在下游执行 DDL 语句失败会有什么表现，如何恢复？
+
+从 v5.0.0 版本开始，如果某条 DDL 语句执行失败，同步任务 (changefeed) 会自动停止，checkpoint-ts 断点时间戳为该条出错 DDL 语句的结束时间戳 (finish-ts) 减去一。如果希望让 TiCDC 在下游重试执行这条 DDL 语句，可以使用 `cdc cli changefeed resume` 恢复同步任务。例如：
```
Comment (yikeke) [inline ticdc/troubleshoot-ticdc.md:378]: @TomShawn  master 上的改动应该改成 5.0 的这个版本：
image

Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:378]: 那似乎可以从这个 PR cp 到 master?

Comment (yikeke) [reply ticdc/troubleshoot-ticdc.md:378]: 也可

**Pull request merged**